spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -1,43 +1,120 @@
 require 'spec_helper'
 
+describe Griddler::Cloudmailin::Adapter do
+  it 'registers itself with griddler' do
+    expect(Griddler.adapter_registry[:cloudmailin]).to be(
+      Griddler::Cloudmailin::Adapter
+    )
+  end
+end
+
 describe Griddler::Cloudmailin::Adapter, '.normalize_params' do
-  it_should_behave_like 'Griddler adapter', :cloudmailin, {
-    envelope: {
-      to: 'Hello World <hi@example.com>',
-      from: 'There <there@example.com>',
-    },
-    plain: 'hi',
-    headers: { Cc: 'emily@example.com' },
-  }
+  context 'without bcc' do
+    it_should_behave_like(
+      'Griddler adapter',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.